cnf-certification-test/networking/suite.go

```diff
@@ -67,8 +80,105 @@ var _ = ginkgo.Describe(common.NetworkingTestKey, func() {
 	ginkgo.It(testID, ginkgo.Label(testID), func() {
 		testMultusNetworkConnectivity(&env, defaultNumPings, netcommons.IPv6)
 	})
+	// Default interface ICMP IPv6 test case
+	testID = identifiers.XformToGinkgoItIdentifier(identifiers.TestListenAndDeclaredIdentifier)
```

Comment:
Please, use this identifier: identifiers.TestUndeclaredContainerPortsUsage, as it's the one it was in the old implementation.